packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/profil/translations/Messages_fr_FR.json

```diff
@@ -1,10 +1,12 @@
 {
-  "xdsl_access_profile_tooltip_interleaved": "<b>Interleaved :</b> Profil auto par défaut",
-  "xdsl_access_profile_tooltip_fast": "<b>Fast : </b> Profil auto en fastPath (ping amélioré sans correction d'erreur)",
-  "xdsl_access_profile_tooltip_ginp": "<b>G.INP : </b> Cette extension améliore la correction d’erreur sans affecter la performance de la ligne. Le DSLAM va utiliser le mode de protection (INP, Impulse Noise Protection) contre le bruit.",
-  "xdsl_access_profile_tooltip_auto": "<b>AUTO-[valeur]M :</b> Le DSLAM va synchroniser au maximum sur le débit",
-  "xdsl_access_profile_tooltip_snr": "<b>SNR-[valeur] : </b> Le DSLAM va analyser la qualité de la ligne et va negocier un rapport signal/bruit (SNR) au chiffre indique. Plus le chiffre est grand plus la stablilité de la ligne sera bonne mais plus le débit sera petit. Sur les profils sans SNR forces, le DSLAM essaie ;d'avoir un SNR de 6.",
   "xdsl_access_profile_tooltip_time": "La modification sera effective sous 1 à 5 minutes.",
   "xdsl_access_dslam_an_error_ocurred": "[DSLAM] Une erreur est survenue !",
-  "xdsl_access_profile_doing": "Changement de profil en cours"
-}
\ No newline at end of file
+  "xdsl_access_profile_doing": "Changement de profil en cours",
+  "xdsl_access_profile_tooltip_adsl_24m": "<b>ADSL_24M :</b> Débit maximum à 24000 / 1024 marge au bruit à 6 dB",
+  "xdsl_access_profile_tooltip_adsl_2m": "<b>ADSL_2M :</b> Débit maximum à 2432 / 1024 marge au bruit supérieur à 6 dB",
+  "xdsl_access_profile_tooltip_adsl_512k": "<b>ADSL_512K :</b> Débit maximum à 608 / 160 marge au bruit supérieur à 6 dB",
+  "xdsl_access_profile_tooltip_adsl_safe1": "<b>ADSL_SAFE1 :</b> Débit maximum à 16128 / 1024 marge au bruit à 10 dB",
+  "xdsl_access_profile_tooltip_adsl_safe2": "<b>ADSL_SAFE2 :</b> Débit maximum à 16128 / 1024 marge au bruit à 16 dB",
+  "xdsl_access_profile_tooltip_adsl_perf1": "<b>ADSL_PERF1 :</b> Débit maximum à 24000 / 1024 marge au bruit à 3 dB",
+  "xdsl_access_profile_tooltip_adsl_perf2": "<b>ADSL_PERF2 :</b> Débit maximum à 24000 / 1024 marge au bruit à 1 dB"
```

Comment:
Can't these values be passed as parameters to the translations ?

Reply:
Yes, it could. But, perhaps, it's a little overenginering, no ?

Reply:
It could in fact avoid us to modify this file for each new value that may exists :)

Reply:
But many if to set the correct values, no ?

Reply:
You can make a translation like:
`"<b>ADSL_PERF2 :</b> Débit maximum à {{ maxFlow }}/ {{ base }} marge au bruit à {{ decibel }} dB"`. And passing parameters through `translate-values`

But it depends on where do you get values, or if it is fully hard coded

Reply:
It's fully hard coded, I just received the list of connection profile from API v6. I parsed the list to display the correct message.

Reply:
Yes but there is no benefit to map the profile list into a new translation key as I have mentioned here: https://github.com/ovh/manager/pull/2990#discussion_r444263968.

But it could be enhance later if needed.